glslc/src/dependency_info.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2015 The Shaderc Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GLSLC_DEPENDENCY_INFO_H
+#define GLSLC_DEPENDENCY_INFO_H
+
+#include <set>
+#include <string>
+
+namespace glslc {
+class DependencyInfoDumpingHandler {
+ public:
+  // Sets -MT argument.
+  void SetDepTargetFileLabel(const std::string& target_label) {
```

Comment:
I'd just call this "SetDepTarget".  

It's simpler still accurate, and more in line with the first sentence of the the GCC doc for -MT
"Change the target of the rule emitted by dependency generation. "
https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html#Preprocessor-Options 

Also, the comment would be more clear if you described what it's doing in English rather than just referencing "-MT".
You can say both. E.g.
"Sets the dependency target explicitly.  It's the same as the argument to -MT"